src/Schema/Events/Authentication.ts

```diff
@@ -160,3 +161,25 @@ export type AuthTrigger = "click" | "timed"
  * the service the user used to authenticate
  */
 export type AuthService = "apple" | "email" | "facebook"
+
+/**
+ * A user provides us with important data during the onboarding flow
+ *
+ * This schema describes events sent to Segment from [[OnboardingUserInputData]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "onboardingUserInputData",
+ *    context_module: "onboardingInterests",
+ *    context_page_owner_type: "onboarding",
+ *    data_input: "[learnAboutArt]"
```

Comment:
Can you clarify what the array syntax in this field is? I also wonder if `data_input` needs a more specific title. We use `subject` in some interfaces to display the text from a clicked item, which seems similar.

Reply:
The issue is that users can choose multiple options in onboarding, so I want to record everything they choose. Would that still be a string? 

Here's an example:
<img width="1440" alt="Screen_Shot_2020-09-17_at_11 29 25_AM" src="https://user-images.githubusercontent.com/12847213/98178210-38cb3280-1eca-11eb-8e4f-8551db570576.png">

Reply:
In that case it would be typed as:
`data_input: string[]`
An example would read as `data_input: ["learnAboutArt"]` 

Regarding the use of strings for the value, do you want the exact text or a shorthand version? `Learn about art` vs `learnAboutArt` -- if we know to expected values we may want to type the allowed strings.

Reply:
Good question - let's keep it as the exact string actually. I'll update it